lib/client/api_matrix.es6.js

```diff
@@ -11,12 +11,13 @@ foam.CLASS({
     to CSV format string or JSON matrix that can be displayed on HTML as
     a nested table.`,
   requires: [
-    'org.chromium.apis.web.Browser',
-    'org.chromium.apis.web.WebInterface',
-    'org.chromium.apis.web.BrowserWebInterfaceJunction',
-    'foam.dao.EasyDAO',
+    'foam.dao.AnonymousSink',
```

Comment:
Yeah... We need more "You should be looking here..." stuff in our READMEs and other places.